Makefile

```diff
@@ -1,9 +1,16 @@
-CC = g++
-CFLAGS =
+CC = clang++
+INCLUDE = -I../ELFIOgit
+CFLAGS = -Wno-initializer-overrides -Wno-c99-designator $(INCLUDE)
 BUILDDIR = Build
 
-all: mkdir
-	$(CC) $(CFLAGS) main.cpp functions.cpp instructions.cpp -o $(BUILDDIR)/simulator
+XCC = riscv64-elf-gcc
+XCFLAGS = -nostdlib -march=rv64g
```

Comment:
isn't it supposed to be `-march=rv64i`?

Reply:
For some reason `riscv64-elf-gcc` says that `addi` is somehow connected with `d` extension
![image](https://github.com/user-attachments/assets/28968329-8ec8-40d8-b434-285a12bc2f1b)

Reply:
Anyway, you're right, fixed `Makefile`